packages/compiler-cli/package.json

```
@@ -13,7 +13,7 @@
     "reflect-metadata": "^0.1.2",
     "minimist": "^1.2.0",
     "canonical-path": "1.0.0",
-    "chokidar": "^1.4.2",
+    "chokidar": "^2.1.0",
```

Why are you using a different chokidar version in **packages/compiler-cli/package.json** as compared to **package.json**?
```suggestion
    "chokidar": "^2.1.1",
```